app/src/main/res/layout/fragment_submission_detail.xml

```diff
@@ -31,7 +31,7 @@
             android:id="@+id/submission_detail_contact_count"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text='@{"Reported contact count: " + Integer.toString(viewModel.selectedSubmission.submission.contactCount)}'
+            android:text="@{@string/submission_detail_contact_count(viewModel.selectedSubmission.submission.contactCount)}"
```

Comment:
All the strings are hardcoded in the xml.
It may be better to move all of them to `strings.xml` to format them and edit them if needed

Reply:
Thanks 👍 Will address updates to the submission detail page including 2-way databinding in a separate branch